shell/platform/windows/fixtures/main.dart

```diff
@@ -97,24 +97,6 @@ void exitTestExit() async {
     closed.complete(data);
```

Comment:
Perhaps rename `closed` to `responded` since the app isn't actually closed by the time `closed` is completed.

Also, consider adding a comment that Windows shutdown is asynchronous so the Dart UI thread may not terminate immediately.